.github/workflows/tests.yml

```
@@ -189,6 +189,32 @@ jobs:
         name: system-test-screenshots
         path: tmp/screenshots/*
 
+  screenshots_link:
+    name: Comment screenshots link
+    #needs: system-tests
```

Once we confirm that the link works... https://github.com/publiclab/plots2/suites/4661607145/artifacts/126679278

```suggestion
    needs: system-tests
```